packages/cli/src/commands/hydrogen/preview.ts

```diff
@@ -35,7 +36,7 @@ export async function runPreview({
 
   const miniOxygen = await startMiniOxygen({
```

Comment:
In the `dev` command we pass environment variables with `getAllEnvironmentVariables`. Should we do that here too? Otherwise, running the `preview` command in any folder will mean Hydrogen won't run because env vars like `SESSION_SECRET` aren't set?